rmgpy/solver/surface.pyx

```diff
@@ -219,6 +231,10 @@ cdef class SurfaceReactor(ReactionSystem):
                               rxn.get_surface_rate_coefficient(self.T.value_si,
                                                                self.surface_site_density.value_si
                                                                ))
+
+                if rxn.kinetics.cov:
+                    self.cov_dep[j] = rxn.kinetics.cov  # dict
```

Comment:
Perhaps we could convert SMILES to species index here (if we choose SMILES in cov dep dict).  Iterate through the SMILES and params in the dict, look up the index for that SMILES, and make a new dict with species_index as keys and params are values.  I think this should be more efficient since we will not have to do SMILES lookups in a dictionary at each timestep to find the coverage.  We could just have array of coverages and find the cov from the index.